src/components/visualizations/chart/highcharts/plugins/zeroAlignYAxis.ts

```diff
@@ -104,7 +106,7 @@ function getRange(targetNumber: number, delta: number): number[] {
     let position = 0;
     const result = [];
 
-    const limit = Math.abs(targetNumber);
+    const limit = Math.round(Math.abs(targetNumber) * DECIMAL) / DECIMAL;
```

Comment:
Have you read this SO post?
https://stackoverflow.com/questions/11832914/round-to-at-most-2-decimal-places-only-if-necessary
you used most upvoted answer but it has some exceptions when it can fail. 
See the second most upvoted answer for different solution

Reply:
Thanks for your suggestion. it's ready to review now :)